### DISCUSSION TITLE
Read-only mode?

### DISCUSSION THREAD

**Discussion by lonix1** [NginxProxyManager/nginx-proxy-manager#3740] [Q&A]
I've always used nginx (within docker) the old-fashioned way, with config files, etc. I prefer this.

But I've also been using traefik for a while, and enjoy its read-only UI. Config is manual, but I have a friendly UI for viewing the status, diagnosing issues, etc.

I'd like to try out NginxProxyManager, and was wondering whether it can also be used in such a read-only mode?

**Comment by paranoidsat** [comment]
Hi
I've stumbled across your question. Maybe this will help others. I'm using podman quadlets

[Container]
...
ReadOnly=yes
DropCapability=all
AddCapability=CAP_NET_BIND_SERVICE CAP_CHOWN CAP_SETUID CAP_SETGID
NoNewPrivileges=false
PidsLimit=100
Tmpfs=/var/cache/nginx:rw,noexec,nosuid,nodev,size=128M
Tmpfs=/var/lib/nginx:rw,noexec,nosuid,nodev,size=128M
Tmpfs=/var/log/nginx:rw,noexec,nosuid,nodev,size=128M
Tmpfs=/var/lib/logrotate:rw,noexec,nosuid,nodev,size=16M
Tmpfs=/etc:rw,nosuid,nodev,size=64M
...

Have fun!
